### ISSUE TITLE
SVG <use> does not display new elements after state update

### ISSUE THREAD

**Issue by mijamo** [facebook/react#5586]
Hello,

I have a component this way :

```
var IconesSVG = React.createClass({
	getInitialState: function() {
		return {icones:{
			icone1: {positions: [[1,56],[-2,57],[-3, 58],[-8, 59]]}
		}}
	},
	componentDidMount:function() {
		var c = this
		setTimeout(function() {
			c.setState({icones:{
			    icone1: {positions: [[2,58],[3,57],[4, 59],[-8, 59], [-11, 60], [-12, 61]]}
		        }})
		}, 4000)
	},

	render: function() {
		var c = this
		var icones = _.keys(c.state.icones).map(function(uuid) {
			return c.state.icones[uuid].positions.map(function(position, index) {
				var pixels = c.props.projection(position)
				return <use key={index} x={pixels[0]} y={pixels[1]} xlinkHref={"#" + uuid} />
			})
		})
		return (
				<g>
					{test}
					{icones}
				</g>
			)
	}
})
```
When the component is mounted everything works as espected : the different locations of <use /> are rendered properly:

But when the state is updated (here after 4 sec for the demonstration), the new elements are not displayed. They are in fact present in the DOM, but not displayed probably due to issues with namespace. When I cut & paste them with firefox tools in the same place, they suddenly appear.

**Comment by mijamo** [comment]
As a complement :
- the elements create at the initial render have the  namespace "http://www.w3.org/2000/svg" and have all the properties of SVG elements (ex: ownerSVGElement)
- The elements created after the initial render have the namespace "http://www.w3.org/1999/xhtml" and do not have any SVG property

**Comment by zpao** [comment]
Yes, `use` needs to be added to a list so that it's created specially. https://github.com/facebook/fbjs/pull/76 exists to address that, though we're planning on fixing SVG more broadly in React itself soon (right @spicyj?)

**Comment by mijamo** [comment]
Thank you for the quick answer, I changed this list it manually in my modules in the meantime and you are right it fixed the issue.

**Comment by spicyj** [comment]
This should be fixed in master already.

**Comment by mijamo** [comment]
Not solved as of React 0.14.6, I still need to manually change fbjs getMarkupWrap to make it work.

**Comment by spicyj** [comment]
Our point releases only have small bug fixes. React v15 will have the fix.
